test/integration/dataContract/DataContractFacade.spec.js

```diff
@@ -50,6 +50,15 @@ describe('DataContractFacade', () => {
 
       expect(result.toJSON()).to.deep.equal(dataContract.toJSON());
     });
+
+    it('should create DataContract from string twice', () => {
```

Comment:
What is the reason behind it?

Reply:
There was a bug in validation, and we had fatal error, if we tried to make it twice in one instance.